### PR TITLE
Add python kaggle course

### PR DESCRIPTION
**Pull request by anjones1000**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -303,7 +303,7 @@
 * [Java Online Training | Edureka](https://www.youtube.com/watch?v=hBh_CC5y8-s) - (Edureka)
 * [Java Programming: Solving Problems with Software](https://www.coursera.org/learn/java-programming) - (Coursera)
 * [Learn how to program: Java](https://www.learnhowtoprogram.com/java-june-2017) - Epicodus Inc.
-* [My Free Java Course - Automation, Development](https://angiejones.tech/free-java-course-test-automation/) - Angie Jones
+* [My Free Java Course - Automation, Development](https://angiejones.tech/free-java-course-test-automation/)  - Angie Jones (Applitools)
```
Comment (SethiPandi) [inline free-courses-en.md:306]: The link you placed here looks more like a blog post pointing to the course material, not to the course itself.
Could we use the link that takes the user directly to the course instead? 

Also, please use the name as given from the source.

```suggestion
* [Java Programming](https://testautomationu.applitools.com/java-programming-course/) - Angie Jones (Applitools)
```

**Review by SethiPandi — COMMENTED** [review]
The change suggested would also require reordering the list.

File: free-courses-en.md (hunk 1 above)
Comment (anjones1000) [reply free-courses-en.md:306]: Absolutely!

Comment (SethiPandi) [reply free-courses-en.md]: Thank you for the fix!
Just a small thing as well, sorry I didn't say it explicitly, I only put it in the suggestion.

Could you remove the extra space around the hyphen? (Before the author's name.)

Comment (anjones1000) [reply free-courses-en.md]: No worries, I am still learning how to use github. I appreciate your feedback.

**Pull request merged**